modules/ting/src/Search/TingSearchFieldFilter.php

```diff
@@ -52,12 +56,17 @@ class TingSearchFieldFilter implements BooleanStatementInterface {
    * TingSearchFieldFilter constructor.
    *
    * TODO BBS-SAL: Consider handling the $operator via an enum.
+   * TODO BBS-SAL: If we end up needing to compare null-values the default for
+   * $value needs to be a
```

Comment:
Something seems to be missing here?

Reply:
Hah, the literally point where I realised I should just go ahead and use a placeholder-value instead of a NULL.